content/docs/04.5-devx/03-cluster-groups/35-ingress-cluster-group.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Set Up Ingress"
+metaTitle: "Set Up Ingress for Cluster Groups"
+metaDescription: "Learn how to configure Ingress for a Palette Cluster Group"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Overview
+
+Cluster Groups may have a cluster endpoint type of either Load Balancer or Ingress. The cluster endpoint type determines how Palette Virtual Clusters deployed in a Cluster Group are exposed. You specify the cluster endpoint in Cluster Group Settings.
+
+Using **Ingress** as the cluster endpoint type is a more cost effective way to access your Kubernetes workloads than using type **Load Balancer**, which requires a new cloud Load Balancer to be provisioned for each virtual cluster.
+
+When you enable **Ingress** as the endpoint for a Cluster Group, you must deploy an Ingress Controller add-on profile, such as NGINX, on each host cluster in the Cluster Group. The Ingress Controller provides the necessary routing functionality for external traffic to reach the Kubernetes API server of each virtual cluster, as well as any apps each virtual cluster contains. 
+
+# Prerequisites
+
+- At least one infrastructure or cloud-based cluster you’ve created in Tenant scope.
+- The Ingress Controller must have Secure Socket Layer (SSL) passthrough enabled so that Transport Layer Security (TLS) is not terminated. <br /><br />
+
+    ```yml
+    extraArgs:        
+  enable-ssl-passthrough: true
+    ```
+
+    Palette provides the ```ingress-nginx-host-cluster``` add-on profile with SSL passthrough already enabled. Palette's NGINX add-on profile also reroutes incoming requests through port 443 to use the cloud Load Balancer. 
+
+- If you are not using Palette's NGINX add-on profile, you can add the following to the profile you're using to reroute incoming requests from port 6443 to port 443. 
+
+    This will terminate TLS for your Apps at the cloud Load Balancer while still allowing you to connect to the API servers of your Virtual Clusters. Alternatively, you must use Cert Manager to issue certificates for App Ingress within the cluster.<br /><br />
```

Comment:
```suggestion
    This will terminate TLS for your Apps at the cloud Load Balancer while still allowing you to connect to the API servers of your Virtual Clusters. Alternatively, you must use a certificate manager, such as the [Cert Manager](/integrations/certmanager) to issue certificates for App Ingress within the cluster.<br /><br />
```
Adding a link to the certs pack, and clarifying that a cert manager will work and does not necessarily have to be the Kubernetes cert manager.